source/_docs/installation.markdown

```diff
@@ -27,7 +27,13 @@ Home Assistant provides multiple ways to be installed. A requirement is that you
     <div class='img-container'>
       <img src='/images/supported_brands/home-assistant.png' />
     </div>
-    <div class='title'>Hass.io<br>(Beginner friendly)</div>
+    <div class='title'>Hass.io on HassOS<br>(Beginner friendly)</div>
+  </a>
+  <a class='option-card' href='/hassio/installation/#alternative-install-on-generic-linux-server'>
```

Comment:
This is something which should go into the "Alternative installs" because that definitely not something that we recommend.

Reply:
That is something the home assistant team decides and that is totally okay.
When I want to add this to the page should I revert my commit and create a new one with this change but in the "Alternative installs" section?

Just do not know the workflow within a big project like this.

Reply:
It is OK to update this PR @ajfriesen, that way this discussion and the end result will stick together.